### DISCUSSION TITLE
Vanilla nvim UI showing when certain errors come up

### DISCUSSION THREAD

**Discussion by finxlfantasy** [LazyVim/LazyVim#1269] [Q&A]
Hey, how do i go back to the normal Lazy.Vim UI, instead of vanilla nvim UI? Is there any way to restore it? 
![image](https://github.com/LazyVim/LazyVim/assets/113019900/4d1f4340-7b81-4977-8a08-27080f5140e8)
 
This is the error that is showing. (Came as a result of trying to add transparency to tokyo-night, which didn't work). My solution up to now has been to clean-install every time this happens lol. 
![image](https://github.com/LazyVim/LazyVim/assets/113019900/2f9c9443-637c-4159-98ba-d86a634f0bbc)

**Comment by dpetka2001** [comment]
You must have overwritten the colorscheme `tokyonight` somewhere in your config and it can't find `tokyonight` any more. Plz start anew and come to a working Lazyvim state. Then add in a file in `~/.config/nvim/lua/plugins/colorscheme.lua` the following
```lua
return {
  {
    "folke/tokyonight.nvim",
    opts = {
      transparent = true,
    },
  },
}
```
Your terminal should also support transparent opacity and be configured to do so.

**Comment by finxlfantasy** [comment]
Thanks, did that but next time is there anything i can do to avoid starting a new nvim config? Btw transparency not working.

**Reply by dpetka2001** [reply]
I used the same configuration above and tokyonight becomes transparent. Must be something in your configuration. Is your terminal also transparent? You have to configure your terminal as well. I use kitty and I have to configure both kitty and tokyonight to have transparency.
